bot_jobs/process_data.js

```diff
@@ -1,112 +0,0 @@
-// Switched to python backend we no longer do this now.
```

Comment:
Why delete this file? So this is already irrelevant? (migrated out from this file even before this PR already?)

Reply:
;-;

Reply:
This has been unused for a long time. So might as well.